content/01.abstract.md

```diff
@@ -1,14 +1,13 @@
 ## Abstract {.page_break_before}
 
-Important tasks in biomedical discovery, such as repurposing drugs or predicting the function of genes and their association to disease, are often framed as network edge prediction problems.
-The number of edges that each node has, termed degree, varies drastically across nodes in real world networks.
-Since degree strongly influences edge prediction, any imbalance or bias in the distribution of degrees leads to nonspecific or misleading predictions.
-We introduce a framework to quantify the effect of node degree on edge prediction.
-Using degree-preserving network permutations, we define an "edge prior" to quantify the probability that two nodes are connected based only on their degree.
-Applying the edge prior to 20 different biomedical networks (16 bipartite, 3 undirected, 1 directed),
-we demonstrate that node degree information alone is capable of remarkable performance, with AUROCs frequently exceeding 85%.
-The predictive performance of degree diminishes on testing networks that were generated using distinct methods and hence show little degree correlation with the training network.
-Accordingly, we suggest using the edge prior as baseline for edge prediction tasks.
-Performance exceeding the baseline is attributable to factors other than degree alone and is often only a small portion of overall performance.
-We provide an open source Python package, named [`xswap`](https://github.com/hetio/xswap) after the permutation algorithm, that allows users to permute their own networks and compute edge priors.
-Our implementation includes an analytic approximation of the edge prior for efficiency and provides a high-level API to accomodate a variety of network and edge types.
+Important tasks in biomedical discovery such as predicting gene functions, gene-disease associations, and drug repurposing opportunities are often framed as network edge prediction.
+The number of edges connecting to a node, termed degree, can vary greatly across nodes in real biomedical networks.
```

Comment:
does "across nodes" really get across what we mean (that within the same network degree varies)? It's a bit redundant IMO because degree implies  "across nodes"

Reply:
I thought this was trying to also introduce differences degree distribution, not just about the individual node degree. If the idea is just to introduce degree, then I think you need a follow-up section somewhere that also notes that the distribution can vary. The next sentence does some of this lifting, but it just says that the distribution can be biased - not that it is also different across networks.